libres/lib/analysis/ies/ies.cpp

```diff
@@ -436,13 +436,13 @@ void ies::linalg_store_active_W(ies::data::Data *data, const matrix_type *W0) {
     int i = 0;
     int j;
     matrix_type *dataW = data->getW();
-    const bool_vector_type *ens_mask = data->ens_mask();
+    const std::vector<bool> ens_mask = data->ens_mask();
```

Comment:
For a little future proofing this can be a reference; due to lifetime extension this will work even though `data->ens_mask()` returns a copy.